src/darwin/Framework/CHIP/MTRDeviceControllerParameters.h

```diff
@@ -77,6 +77,11 @@ MTR_AVAILABLE(ios(17.6), macos(14.6), watchos(10.6), tvos(17.6))
  */
 - (void)setOTAProviderDelegate:(id<MTROTAProviderDelegate>)otaProviderDelegate queue:(dispatch_queue_t)queue;
 
+/**
+ * Sets the maximum subscriptions allowed for devices on Thread. This defaults to 3.
```

Comment:
1. This makes it sound like we can only subscribe to at most 3 Thread devices.  Needs to be rephrased to make it clear what this is really controlling.
2. The behavior of setting this to 0 needs to be documented.
3. Do we want to make this thread-specific?  Or make it more generic in some way where this is the cap on the parallel subscriptions on low-bandwidth links and we get to decide internally that "low-bandwidth" means "Thread"?

Reply:
Will update the comments to be clearer and make it clamp to minimum of 1.

Reply:
I guess the question is whether we want a "use whatever the default is, even if it changes later" value.  Right now 0 is that value, effectively....  Which is fine, but that should just be documented.  At least if we want to support it.

Reply:
I've changed it so that the parameter object initializes it to a default value. And if it's forcibly changed to 0, the real value is clamped to 1, as documented.